client/src/components/Jobs/Jobs.jsx

```diff
@@ -133,15 +134,25 @@ function JobsView({
     {
       Header: 'Job Name',
       accessor: 'name',
-      Cell: (el) => (
-        <span
-          title={el.value}
-          id={`jobID${el.row.index}`}
-          className="job__name"
-        >
-          {el.value}
-        </span>
-      ),
+      Cell: (el) => {
+        const query = queryStringParser.parse(useLocation().search);
```

Comment:
Hmm, I wonder if there is a better way to get the query string value without having to query for it in every cell function. A potential solution if you can try it would be to wrap the columns in a useMemo and have the query.query_string as a dependency. I'm not sure if that would work but could be worth a try

Reply:
This has been fixed. Thank you for suggesting.